### ISSUE TITLE
Additional annotation for 2,6-diaminopurine (CHEBI:38001)

### ISSUE THREAD

**Issue by cviner** [ebi-chebi/ChEBI#3183]
It would be useful if the below details could be added to the [2,6-diaminopurine (CHEBI:38001)](http://www.ebi.ac.uk/chebi/searchId.do;?chebiId=CHEBI:38001) entry.

Synonym: 2-aminoadenine

A ![has functional parent](https://camo.githubusercontent.com/a2db830ba48121b41b8203ac2ecda097789e04ed/687474703a2f2f7777772e6562692e61632e756b2f63686562692f696d616765732f6f6e746f6c6f67792f636865636b65642f6861735f66756e6374696f6e616c5f706172656e742e676966) has functional parent relationship with [adenine (CHEBI:16708)](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16708) should be added.

Chemical details, including SMILES can be imported from its PubChem entry: https://pubchem.ncbi.nlm.nih.gov/compound/30976

The below references could be added:

- [Kirnos, M., Khudyakov, I., Alexandrushkina, N., & Vanyushin, B. (1977). 2-Aminoadenine is an adenine substituting for a base in S-2L cyanophage DNA. _Nature_, 270(5635), 369-370.](http://doi.org/10.1038/270369a0)
- [Khudyakov, I., Kirnos, M., Alexandrushkina, N., & Vanyushin, B. (1978). Cyanophage S-2L contains DNA with 2,6-diaminopurine substituted for adenine. _Virology_, 88(1), 8-18.](http://doi.org/10.1016/0042-6822(78)90104-6)

**Comment by G-Owen** [comment]
Hi Coby,

CHEBI:38001 is actually a class name rather than an individual compound - I have added an 's' to the end of the name to make it clearer. It doesn't have a structure as the term can include various tautomers (double bond/position of the N-H, etc.) as well as substituents at any position, so there is no InCHI, SMILES, etc.

The entry you probably want is 9H-purine-2,6-diamine (CHEBI:40235), which I have now updated to include the references and functional parent relationship you mention.

Best regards,
Gareth
